includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -140,7 +144,11 @@ public function should_use_sensei_theme() {
 	 *
 	 * @return bool
 	 */
-	public function has_sensei_theme_enabled( int $course_id ) {
+	public static function has_sensei_theme_enabled( int $course_id ) {
```

Comment:
I think we can take the opportunity of this PR, and also add a filter here for users that want to create exceptions here. Could we add it in this PR?

Just notice that since we're using the "learning mode" copy now, it would be nice to use the filter with this copy too.

Reply:
Good point! Added a `sensei_has_learning_mode_enabled` hook. 8e6686334743c191ddf589e2dd0f4bc9318e0bd2

Reply:
I think you also added a `use phpDocumentor\Reflection\Types\Boolean;` as part of this PR. Was it accidental or on purpose?

Also, I think something like `sensei_course_learning_mode_enabled` would be a little more intuitive for who is reading the filters. Although the method is `has_sensei_theme_enabled` (a check), the purpose while using the filter would be to enable the learning mode. Does it make sense? cc @donnapep since she's thinking about hook names standard. Maybe she has an insight into that.

Also, it would be nice to add the hook to the PR description too. ;)

Reply:
> I think you also added a use phpDocumentor\Reflection\Types\Boolean; as part of this PR. Was it accidental or on purpose?

Nope. Probably my editor auto imported it and I didn't notice. Thanks for spotting it!

> Also, I think something like sensei_course_learning_mode_enabled would be a little more intuitive for who is reading the filters. Although the method is has_sensei_theme_enabled (a check), the purpose while using the filter would be to enable the learning mode. Does it make sense? cc @donnapep since she's thinking about hook names standard. Maybe she has an insight into that.

Also, it would be nice to add the hook to the PR description too. ;)
Updated the name to `sensei_course_learning_mode_enabled`. ade117ee8d8024a2844777e1255df8c7942cff5b. Also updated the description.